documentation/API_Release_Guidelines.md

```diff
@@ -65,68 +83,75 @@ API release numbers are GitHub tags of the format "rx.y".
+**Create the release**
+* manage the release PR approval
+* merge the approved release PR
+* create the release:
+  * an API release is created using the GitHub release feature (a release tag and, optionally, a release package).
+  * the release name shall be the same as the release tag and shall have the following format: rx.y
+  * the x.y number shall follow the release numbering scheme  as defined in the above section on API release numbering
+  * note: outside the API Sub Project, the release name shall be referred to with the API name (for definition see the versioning section in the [API Design Guidelines](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md)) followed by the release number e.g. quality-on-demand rx.y
+* update the API release tracker with the date and release tag link for the release
 
-**Release process**
+**Maintenance release**
 
-Repeat the above release steps for
+In case a patch update of a public API version x.y.z is required, the patched public API version x.y.z+1 shall be created through a maintenance release on a separate branch referred to as a maintenance branch. 
 
-* alpha releases up to the M3 milestone (at M3 the first release-candidate shall be ready)
-* release-candidates between the M3 and the M4 milestone,
-* at M4, provide the public-release for inclusion in the meta-release at the M5 milestone
+NOTE: a patch is the only case for which a separate branch is created and maintained within the API repository (as pull requests should be prepared within forks of the API repository, c.f. [Governance/CONTRIBUTING.md](https://github.com/camaraproject/Governance/blob/main/CONTRIBUTING.md))
 
-**Release maintenance**
+## Example of the API release process
 
-In case a patch update of a public-release API version x.y.z is required, the patched public-release API version x.y.z+1 shall be created as a maintenance-release on a separate branch referred to as a maintenance branch. 
+To release a MINOR update of a public API version 1.0.0, resulting in the release of public API version 1.1.0:
 
-NOTE: a patch is the only case for which a separate branch is created and maintained within the API repository (as pull requests should be prepared within forks of the API repository, c.f. https://github.com/camaraproject/Governance/blob/main/CONTRIBUTING.md)
+* Develop the 1.1.0 updates on the main branch. The first PR shall update the OAS file setting the API version to wip, and the URL to vwip.
+* Once sufficiently stable, create a release PR for the API version 1.1.0-alpha.1.
+* After release PR approval, create the pre-release rx.1 and publish it on the API release tracker.
+* Several alpha API versions 1.1.0-alpha.p may be released, each setting the API version back to "wip" in the first API update PR and evolving the alpha number with each following pre-release (rx.2 - rx.m).
```

Comment:
"Several" sounds a lot. And not the API is setting the API version back to "wip" ...

```suggestion
* Additional alpha API versions 1.1.0-alpha.p may be released. For each setting the API version back to "wip" in the first API update PR and setting the new version API only within the pre-release. Evolving the alpha number with each following pre-release (rx.2 - rx.m).
```